client/client.go

```diff
@@ -1140,6 +1437,50 @@ func (c *Client) runHandleInvocation(msg *wamp.Invocation) {
 		return
 	}
 
+	// Check and handle Payload PassThru Mode
```

Comment:
btw. should we check if route callee role supports ppt?

Reply:
That is checked on the `router` side on processing `CALL` message, here is `INVOCATION` part. And in general cases there should be no error here. But I'm not sure about all WAMP routers :)